### ISSUE TITLE
[Docker] Crash with latest commit in non interactive mode

### ISSUE THREAD

**Issue by fl4shforward** [prof79/fansly-downloader-ng#43]
## Bug Description
Crash when running in non interactive mode with latest version.

I did put the checkKey entry in the config file by hand to avoid the "Press enter" prompt.

## Actual behavior

Log
```
Fansly Downloader NG v0.9.3
  ███████╗ █████╗ ███╗   ██╗███████╗██╗  ██╗   ██╗    ███╗   ██╗███████╗     █████╗ ██████╗ ██████╗ 
  ██╔════╝██╔══██╗████╗  ██║██╔════╝██║  ╚██╗ ██╔╝    ████╗  ██║██╔════╝    ██╔══██╗██╔══██╗██╔══██╗
  █████╗  ███████║██╔██╗ ██║███████╗██║   ╚████╔╝     ██╔██╗ ██║██║ ███╗    ███████║██████╔╝██████╔╝
  ██╔══╝  ██╔══██║██║╚██╗██║╚════██║██║    ╚██╔╝      ██║╚██╗██║██║  ██║    ██╔══██║██╔═══╝ ██╔═══╝ 
  ██║     ██║  ██║██║ ╚████║███████║███████╗██║       ██║ ╚████║███████║    ██║  ██║██║     ██║     
  ╚═╝     ╚═╝  ╚═╝╚═╝  ╚═══╝╚══════╝╚══════╝╚═╝       ╚═╝  ╚═══╝╚══════╝    ╚═╝  ╚═╝╚═╝     ╚═╝     
                        developed on github.com/prof79/fansly-downloader-ng
                                               v0.9.3
 Info | 19:11 || Reading config.ini file ...
 Info | 19:11 || Name validation for @xxxx successful!
 WARNING | 19:11 || !!! FANSLY MAY BAN YOU FOR USING THIS SOFTWARE, BE WARNED !!!
 WARNING | 19:11 || Make sure, checking the main.js sources of the Fansly homepage, 
                    that the `this.checkKey_` value is identical to this 
                    (text within the single quotes only): `negwij-zyZnek-wavje1`
Press <ENTER> to attempt to continue ...
 ERROR | 19:11 || An unexpected error occurred: EOF when reading a line
Traceback (most recent call last):
  File "/usr/src/fansly-ng/fansly_downloader_ng.py", line 198, in <module>
    exit_code = main(config)
                ^^^^^^^^^^^^
  File "/usr/src/fansly-ng/fansly_downloader_ng.py", line 108, in main
    validate_adjust_config(config)
  File "/usr/src/fansly-ng/config/validation.py", line 427, in validate_adjust_config
    validate_adjust_check_key(config)
  File "/usr/src/fansly-ng/config/validation.py", line 340, in validate_adjust_check_key
    input_enter_continue()
  File "/usr/src/fansly-ng/textio/textio.py", line 98, in input_enter_continue
    input('\nPress <ENTER> to attempt to continue ...')
EOFError: EOF when reading a line
Exiting in 15 seconds ...
```

Config file
```
[TargetedCreator]
username = xxxx

[MyAccount]
authorization_token = xxxx
user_agent = xxxx
check_key = negwij-zyZnek-wavje1

[Options]
download_mode = Normal
show_downloads = True
download_media_previews = False
open_folder_when_finished = False
download_directory = downloads
separate_messages = True
separate_previews = True
separate_timeline = True
metadata_handling = Advanced
use_duplicate_threshold = True
use_folder_suffix = False
interactive = False
prompt_on_exit = False
timeline_retries = 1
timeline_delay_seconds = 60
show_skipped_downloads = True

[Cache]
device_id = xxxxx
device_id_timestamp = xxxxx
```

## Environment Information
 - Operating System: Docker (Linux)
 - Fansly Downloader Version: v0.9.3
 - Fansly Downloader Type: Python

## User Research

I have done the following:
- [x] Looked through [old GitHub issues](https://github.com/prof79/fansly-downloader-ng/issues?&q=is%3Aissue)
- [x] Read the [config.ini options](https://github.com/prof79/fansly-downloader-ng/wiki/Fansly-Downloader-NG-Settings#explanation-of-configini)
- [x] Searched the Web (like [DuckDuckGo](https://duckduckgo.com))

**Comment by prof79** [comment]
Sorry, I certainly f* this up being defensive/cautious not handling non-interactive mode properly. Will fix ASAP.

**Comment by Joly0** [comment]
Just out of curiosity, which docker are you using?

**Comment by fl4shforward** [comment]
> Just out of curiosity, which docker are you using?

My own [fl4shforward/fansly-ng-docker](https://github.com/fl4shforward/fansly-ng-docker)

**Comment by prof79** [comment]
Fixed v0.9.5 onwards https://github.com/prof79/fansly-downloader-ng/releases/tag/v0.9.7
but beware of their new check key countermeasure!

**Comment by fl4shforward** [comment]
Fixed by https://github.com/prof79/fansly-downloader-ng/commit/4739155c687ab06a90adb38c081a764ea24b2d2b
